i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
Agree. However, I don't think we should even do that. In fact this script does not really belong to this repository. It's something anyone who creates a repository will/might need.